components/MapScreen.tsx

```diff
@@ -31,52 +40,68 @@ export default function MapScreen({ navigation }) {
   return (
     <View style={styles.container}>
       <MapView
-        style={styles.map}
-        provider={'google'}
+        style={[styles.map, { height: showVoteBar ? "80%" : "100%" }]}
+        provider={"google"}
         showsUserLocation
         showsMyLocationButton
         showsCompass
       >
         {hospitals.map((hospital) => (
           <Marker
             key={hospital.id}
+            tracksViewChanges={false}
```

Comment:
This reduces the number of event listener per marker and it increases the performance